tinygrad/features/jit.py

```diff
@@ -91,7 +91,7 @@ def __get__(self, obj, objtype): return functools.partial(self.__call__, obj)
 
   def __call__(self, *args, **kwargs) -> ReturnType:
     # all inputs (except const) are realized
-    input_tensors: Dict[Union[int, str], Union[LazyBuffer, MultiLazyBuffer]] = { cast(Union[int, str], k):v.realize().lazydata for k,v in itertools.chain(enumerate(args), kwargs.items()) if v.__class__ is Tensor }  # noqa: E501
+    input_tensors: Dict[Union[int, str], Union[LazyBuffer, MultiLazyBuffer]] = { cast(Union[int, str], k):v.realize().lazydata for k,v in itertools.chain(enumerate(args), sorted(kwargs.items(), key=lambda x: str(x[0]))) if v.__class__ is Tensor }  # noqa: E501
```

Comment:
Do you need to specify this `key=` function? I think it's the same default behavior without specifying the `key`

Reply:
No, `key` and `reverse` are kwarg-only parameters to sorted:
https://stackoverflow.com/questions/75761879/python-sorted-expected-1-argument-got-3-trying-to-understand-named-argument

Reply:
i mean just `sorted(kwargs.items())`